duckbot.py

```diff
@@ -1,6 +1,5 @@
 import logging
-import sys
-import time
+import sys, time
```

Comment:
Same thing as above :)

```suggestion
import sys
import time
```

Reply:
Please apply this too :)